packages/preferences/src/browser/util/preference-tree-generator.ts

```diff
@@ -148,7 +148,7 @@ export class PreferenceTreeGenerator {
         if (!property.owner) {
             return;
         }
-        const groupID = this.defaultTopLevelCategory;
+        const groupID = 'extensions';
```

Comment:
I'm not really sure what this change entails and if someone would ever change the `defaultTopLevelCategory` but if someone this, this change makes it a bit harder for them to get their previous behavior back. Would it at least make sense to add this as an optional argument with default value to the method signature, so that it can be more easily overwritten?